internal/cmd/index_test.go

```diff
@@ -11,12 +11,13 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestIndex(t *testing.T) {
+// TODO flaky tests when ran locally
```

Comment:
Is there anything we can do that will make it less flaky?

Reply:
we couldn't find out the core reason so spun up a ticket attached to the ticket to look at this on as it passes on the second or third go locally